Appraisals

```diff
@@ -1,10 +1,22 @@
 appraise "default" do
 end
 
-appraise "rails_3" do
-  gem "rails", "3.2.19", require: true
+appraise "rails_5_sidekiq_5" do
```

Comment:
We can potentially stop supporting rails 5 soon but for now I think we'll keep it around.